homeassistant/components/nuki/config_flow.py

```diff
@@ -41,7 +41,7 @@ async def validate_input(hass, data):
             DEFAULT_TIMEOUT,
         )
 
-        info = bridge.info()
+        info = await hass.async_add_executor_job(bridge.info)
```

Comment:
A future improvement is to create a function that does both sync calls, ie create the bridge and get the bridge info, and then schedule that function once on the executor. Switching context is expensive.